package.json

```diff
@@ -5,6 +5,8 @@
   "dependencies": {
     "@sentry/browser": "^5.5.0",
     "axios": "^0.19.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
```

Comment:
In your PR description can you add more of an explanation on why you decided to go with Enzyme here instead of something like react-test-renderer?

Reply:
I like enzyme's method find() and the overall style of it. Since we're using airbnb styles anyway hehe